openhcl/underhill_core/src/emuplat/netvsp.rs

```diff
@@ -966,6 +1038,29 @@ impl HclNetworkVFManager {
         ))
     }
 
+    pub async fn save(&self) -> Option<ManaSavedState> {
+        let save_state = self
+            .shared_state
+            .worker_channel
+            .call(HclNetworkVfManagerMessage::SaveState, ())
+            .await;
+
+        match save_state {
+            Ok(None) => {
+                tracing::warn!("No MANA device present when saving state, returning None");
```

Comment:
Is this statement always true? This can also happen if the save state RPC failed in `device.save()`, right?